lib/authlogic/controller_adapters/abstract_adapter.rb

```diff
@@ -58,6 +58,10 @@ def last_request_update_allowed?
         controller.send(:last_request_update_allowed?)
       end
 
+      def respond_to_missing?(*args)
+        super(*args) || controller.respond_to?(*args)
+      end
```

Comment:
chose `respond_to?` instead of `respond_to_missing?` so `method(:...)` works too, h/t @stephencelis